site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -167,7 +174,7 @@ export const WorkspacesPageView = ({
               </MoreMenuContent>
             </MoreMenu>
           </>
-        ) : (
+        ) : invalidPageNumber ? null : (
```

Comment:
this small branch seems really easy to lose in the middle of this giant conditional. it also looks like this will result in an empty `<TableToolbar>` being rendered, which we might as well just not render.

given all of this, might I propose replacing this bit with:

```tsx
{!invalidPageNumber && <TableToolbar>
  {...}
</TableToolbar>}
```